src/vanilla/shallow.ts

```diff
@@ -37,10 +37,10 @@ export function shallow<T>(objA: T, objB: T) {
   if (keysA.length !== Object.keys(objB).length) {
     return false
   }
-  for (let i = 0; i < keysA.length; i++) {
+  for (const keyA of keysA) {
```

Comment:
We generally don't use for-of.
Oh, but we already use it in L28! Failed to review back then. Maybe we should change L28 and L17 with for-let.
And, we can use for-of from v5.
What do you think? @dbritto-dev

Reply:
Why don't you use it? It's great for iterables. 

Why from v5? Is it IE11 again?

Reply:
@dai-shi yeah, same code but let instead of const. btw, we can merge it on v4 and then on v5

Reply:
Yes, we try _best_ to make v4 work on ie11.
Note that es2018 is esm target, but for cjs we target es5 maybe. I'm not sure if everything is transpiled into es5.

Reply:
Wait, `for-of` is properly transpiled. So, this is fine.